src/main/java/uk/gov/hmcts/reform/prl/services/tab/alltabs/AllTabServiceImpl.java

```diff
@@ -115,4 +124,16 @@ public Map<String, Object> getAllTabsFields(CaseData caseData) {
         return getCombinedMap(caseData);
     }
 
+    public void updatePartyDetailsForNoc(CaseData caseData, Optional<SolicitorRole> solicitorRole) {
+        if (caseData != null) {
+            Map<String, Object> caseDataUpdatedMap = new HashMap<>();
+            if (solicitorRole.isPresent() && RESPONDENT.equals(
+                solicitorRole.get().getRepresenting()) && PrlAppsConstants.C100_CASE_TYPE.equalsIgnoreCase(caseData.getCaseTypeOfApplication())) {
+                caseDataUpdatedMap.put(RESPONDENTS, caseData.getRespondents());
+                caseDataUpdatedMap.put(RESPONDENT_TABLE, applicationsTabService.getRespondentsTable(caseData));
+            }
+            refreshCcdUsingEvent(caseData, caseDataUpdatedMap);
```

Comment:
this method will go away following ccd concurrency changes. we will bring a ticket for next sprint